<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>844</width>
    <height>609</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="zoneImage">
     <property name="minimumSize">
      <size>
       <width>640</width>
       <height>480</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="SliderAvecLabel" name="sliderHmin" native="true"/>
     </item>
     <item>
      <widget class="SliderAvecLabel" name="sliderHmax" native="true"/>
     </item>
     <item>
      <widget class="SliderAvecLabel" name="sliderSmin" native="true"/>
     </item>
     <item>
      <widget class="SliderAvecLabel" name="sliderSmax" native="true"/>
     </item>
     <item>
      <widget class="SliderAvecLabel" name="sliderVmin" native="true"/>
     </item>
     <item>
      <widget class="SliderAvecLabel" name="sliderVmax" native="true"/>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnSauverFichier">
       <property name="text">
        <string>Sauver fichier</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SliderAvecLabel</class>
   <extends>QWidget</extends>
   <header>sliderAvecLabel.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
